rtcpeerconnection_test.go

```diff
@@ -308,6 +308,33 @@ func TestSetRemoteDescription(t *testing.T) {
 	}
 }
 
+func TestCreateOfferAnswer(t *testing.T) {
+	OfferPeerConn, err := New(RTCConfiguration{})
```

Comment:
Why are these local variables capitalized? It makes them seem like globals.

Reply:
Noted, i'll change it.